docs/sphinx_source/conf.py

```diff
@@ -77,7 +77,12 @@ def configureDoxyfile(input_dir, output_dir):
 # a list of builtin themes.
 #
 html_theme = "sphinx_rtd_theme"
-
+html_logo = "../_static/crown_logo_bw.svg"
```

Comment:
I wouldn't commit any directory called `_static` since it hints to a temporary folder. Why not calling it `logos` and link there?